### PR TITLE
Change error messages there are still issues with IE and FF with screen readers

### PR DESCRIPTION
**Pull request by deniselashlley**
- Changed the error message so that they will re-read the messages again in Firefox
- Changed the postcode lookup to be totally hidden so that the fields as not read out, don't allow the ability to tab hidden fields.
- Added a check to reset country selection on re-selection

### CODE REVIEW

File: src/components/SelectField/SelectField.js [hunk 1]
```diff
@@ -144,7 +144,7 @@ class SelectField extends Component {
     const errorClass = this.state.showErrorMessage === true ? 'form__field-error-wrapper' : '';
     const extraClass = this.props.extraClass !== '' ? this.props.extraClass : '';
     const isBrowser = browser();
-    const supportedAriaAttributes = isBrowser.name === 'firefox' && isBrowser.os.match('Windows') ? { role: 'alert', 'aria-relevant': 'all' } : { role: 'status' };
+    const supportedAriaAttributes = isBrowser.name === 'firefox' && isBrowser.os.match('Windows') ? { 'aria-live': 'assertive', 'aria-relevant': 'additions removals' } : { 'aria-live': 'assertive', role: 'status' };
```
Comment (Heleen-cr) [inline src/components/SelectField/SelectField.js]: @AndyEPhipps I think we might need to add this to the radiobuttons error message too right?

File: src/components/PostcodeLookup/PostcodeLookup.js [hunk 2]
```diff
@@ -334,6 +337,7 @@ class PostcodeLookup extends Component {
    * @return {*}
    */
   render() {
+    // console.log(this.state.isHidden);
     const postCodeField = {
```
Comment (Heleen-cr) [inline src/components/PostcodeLookup/PostcodeLookup.js]: just a reminder that there's a comment out line here.

Comment (deniselashlley) [reply src/components/PostcodeLookup/PostcodeLookup.js]: oops that shouldnt be there

**Pull request merged**